src/classes/UTIL_RecordTypeSettingsUpdate.cls

```diff
@@ -261,42 +263,46 @@ global virtual class UTIL_RecordTypeSettingsUpdate {
         }
 
         List<String> names = nameString.split(';', 0);
-
-        Set<String> ids = new Set<String>();
-        for (String name : names) {
-            if (isId(name)) {
-                ids.add(name);
-            } else if (nameMap.containsKey(name)) {
-                ids.add(String.valueOf(nameMap.get(name)));
-            }
-        }
-
-        String idsString = String.join(new List<String>(ids), ';');
-
-        return idsString;
+        return resolveNamesToIds(names, nameMap);
     }
 
+    @TestVisible
     /*******************************************************************************************************
     * @description Determines the Record Type Id based on the name
      * @param name A Record Type name
-     * @param rtNamesById A mapping of Record Type names to Record Type Ids
+     * @param nameMap A mapping of Record Type names to Record Type Ids
      * @return String An Id if the name is a valid Record Type name, otherwise null.
     */
-    private String getIdFromName(String name, Map<String, Id> rtNamesById) {
-        if (String.isBlank(name)) {
+    private String getIdFromName(String name, Map<String, Id> nameMap) {
+        if (name == null) {
```

Comment:
why not continue to use isBlank()?  did the code previously handle ignoring spaces between semicolons that now it won't?